packages/react/src/views/EmbeddedChat.js

```diff
@@ -82,18 +100,23 @@ const EmbeddedChat = ({
 
   const [RCInstance, setRCInstance] = useState(() => initializeRCInstance());
 
+  const reInstantiate = useCallback(() => {
```

Comment:
why do we need to re-instantiate?

Reply:
It was there earlier as well, but it was unstructured and caused unnecessary renders. I just fixed some linting issues, that's it. The reason why it is there is because, if someone changes the host, auth, or roomId from Storybook controls, it has to re-establish the connection with that roomId or a new host. If you look at the dependency array, it only includes host, roomId, and auth.